Sprint-2/debug/2.js

```diff
@@ -12,3 +12,6 @@ console.log(`The last digit of 806 is ${getLastDigit(806)}`);
 
 // This program should tell the user the last digit of each number.
 // Explain why getLastDigit is not working properly - correct the problem
+
+// num is declared as a constant variable on ln 3 and all results from then on are based on this value.
+// I have changed the function by having num as an argument in the function.
```

Comment:
The change is good. 

With the way you changed the function, we can also keep `const num=103;` untouched. If you are interested in understanding why, may I suggest looking up (or ask ChatGPT)  "How local variable shadow global variable in JS".

Reply:
Yeah, I've got a good understanding of scope now.